primitives/settings/src/lib.rs

```diff
@@ -20,63 +20,71 @@
-	pub static SIGNING_KEY_FILE: &str = "enclave-signing-pubkey.bin";
-
-	// used by enclave
-	pub const RSA3072_SEALED_KEY_FILE: &str = "rsa3072_key_sealed.bin";
-	pub const SEALED_SIGNER_SEED_FILE: &str = "ed25519_key_sealed.bin";
-	pub const AES_KEY_FILE_AND_INIT_V: &str = "aes_key_sealed.bin";
-	pub const CHAIN_RELAY_DB: &str = "chain_relay_db.bin";
-
-	pub const RA_DUMP_CERT_DER_FILE: &str = "ra_dump_cert.der";
-
-	// used by worker and enclave
-	pub const SHARDS_PATH: &str = "./shards";
-	pub const ENCRYPTED_STATE_FILE: &str = "state.bin";
-
-	#[cfg(feature = "production")]
-	pub static RA_SPID_FILE: &str = "spid_production.txt";
-	#[cfg(feature = "production")]
-	pub static RA_API_KEY_FILE: &str = "key_production.txt";
-
-	#[cfg(not(feature = "production"))]
-	pub static RA_SPID_FILE: &str = "spid.txt";
-	#[cfg(not(feature = "production"))]
-	pub static RA_API_KEY_FILE: &str = "key.txt";
+    // used by worker
+    pub static ENCLAVE_TOKEN: &str = "enclave.token";
+    pub static ENCLAVE_FILE: &str = "enclave.signed.so";
+    pub static SHIELDING_KEY_FILE: &str = "enclave-shielding-pubkey.json";
+    pub static SIGNING_KEY_FILE: &str = "enclave-signing-pubkey.bin";
+
+    // used by enclave
+    pub const RSA3072_SEALED_KEY_FILE: &str = "rsa3072_key_sealed.bin";
+    pub const SEALED_SIGNER_SEED_FILE: &str = "ed25519_key_sealed.bin";
+    pub const AES_KEY_FILE_AND_INIT_V: &str = "aes_key_sealed.bin";
+    pub const CHAIN_RELAY_DB: &str = "chain_relay_db.bin";
+
+    pub const RA_DUMP_CERT_DER_FILE: &str = "ra_dump_cert.der";
+
+    // used by worker and enclave
+    pub const SHARDS_PATH: &str = "./shards";
+    pub const ENCRYPTED_STATE_FILE: &str = "state.bin";
+
+    #[cfg(feature = "production")]
+    pub static RA_SPID_FILE: &str = "spid_production.txt";
+    #[cfg(feature = "production")]
+    pub static RA_API_KEY_FILE: &str = "key_production.txt";
+
+    #[cfg(not(feature = "production"))]
+    pub static RA_SPID_FILE: &str = "spid.txt";
+    #[cfg(not(feature = "production"))]
+    pub static RA_API_KEY_FILE: &str = "key.txt";
 }
 
 /// Settings concerning the worker
 pub mod worker {
-	// the maximum size of any extrinsic that the enclave will ever generate in B
-	pub static EXTRINSIC_MAX_SIZE: usize = 4196;
-	// the maximum size of a value that will be queried from the state in B
-	pub static STATE_VALUE_MAX_SIZE: usize = 1024;
+    // the maximum size of any extrinsic that the enclave will ever generate in B
+    pub const EXTRINSIC_MAX_SIZE: usize = 4196;
+    // the maximum size of a value that will be queried from the state in B
+    pub const STATE_VALUE_MAX_SIZE: usize = 1024;
+    // the maximum size of the header
+    pub const HEADER_MAX_SIZE: usize = 200;
+    // maximum size of shielding key
+    pub const SHIELDING_KEY_SIZE: usize = 8192;
+    // maximum size of signing key
+    pub const SIGNING_KEY_SIZE: usize = 32;
+    // size of the MR enclave
+    pub const MR_ENCLAVE_SIZE: usize = 32;
```

Comment:
added these fixed array sizes to replace the magic numbers inside the functions. In addition, make them `const` instead of `static` (which is mutable)